### PR TITLE
⛧ migrate to nextcloud-webpack-config

### PR DESCRIPTION
**Pull request by blizzz**
it builds. it runs locally. 🤘

for #483

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,42 +28,28 @@
...
 		"node": ">=10.0.0"
 	},
 	"devDependencies": {
 		"@babel/core": "^7.14.5",
+		"@babel/eslint-parser": "^7.14.5",
 		"@babel/plugin-syntax-dynamic-import": "^7.8.3",
 		"@babel/preset-env": "^7.14.5",
+		"@nextcloud/browserslist-config": "^2.1.0",
+		"@nextcloud/eslint-config": "^5.1.0",
+		"@nextcloud/eslint-plugin": "^2.0.0",
```
Comment (skjnldsv) [inline package.json:43]: Included in `@nextcloud/eslint-config`

File: package.json [hunk 2]
```diff
@@ -27,42 +28,28 @@
...
+		"extends @nextcloud/browserslist-config"
 	],
 	"engines": {
 		"node": ">=10.0.0"
 	},
 	"devDependencies": {
 		"@babel/core": "^7.14.5",
+		"@babel/eslint-parser": "^7.14.5",
 		"@babel/plugin-syntax-dynamic-import": "^7.8.3",
 		"@babel/preset-env": "^7.14.5",
```
Comment (skjnldsv) [inline package.json:40]: Included in `@nextcloud/babel-config`

File: package.json [hunk 3]
```diff
@@ -27,42 +28,28 @@
...
-		"eslint-import-resolver-webpack": "^0.13.1",
-		"eslint-loader": "^3.0.4",
-		"eslint-plugin-import": "^2.23.4",
-		"eslint-plugin-nextcloud": "^0.3.0",
-		"eslint-plugin-node": "^10.0.0",
-		"eslint-plugin-promise": "^4.3.1",
-		"eslint-plugin-standard": "^4.1.0",
-		"eslint-plugin-vue": "^5.2.3",
 		"node-sass": "^6.0.0",
 		"sass-loader": "^10.1.1",
```
Comment (skjnldsv) [inline package.json:48]: Included in `@nextcloud/webpack-vue-config`

File: package.json [hunk 4]
```diff
@@ -27,42 +28,28 @@
...
-		"style-loader": "^2.0.0",
-		"stylelint": "^8.4.0",
-		"stylelint-config-recommended-scss": "^3.3.0",
+		"stylelint-config-recommended-scss": "^4.2.0",
 		"stylelint-scss": "^3.19.0",
```
Comment (skjnldsv) [inline package.json:50]: Included in `@nextcloud/stylelint-config`

File: package.json [hunk 5]
```diff
@@ -27,42 +28,28 @@
...
-		"stylelint-webpack-plugin": "^0.10.5",
 		"vue-loader": "^15.9.7",
 		"vue-template-compiler": "^2.6.14",
-		"webpack": "^4.46.0",
-		"webpack-cli": "^4.7.2",
-		"webpack-merge": "^5.8.0"
+		"webpack": "^5.38.1"
```
Comment (skjnldsv) [inline package.json:53]: Included in `@nextcloud/webpack-vue-config`

**Review by skjnldsv — CHANGES_REQUESTED** [review]
See https://github.com/nextcloud/app-tutorial/pull/319

**Comment by blizzz** [comment]
@skjnldsv 
> Run npm ci
  npm ci
  npm run build --if-present
  shell: /usr/bin/bash -e {0}
>npm ERR! fsevents not accessible from chokidar
>
>npm ERR! A complete log of this run can be found in:
npm ERR!     /home/runner/.npm/_logs/2021-06-14T12_22_23_122Z-debug.log
Error: Process completed with exit code 1.

**Comment by blizzz** [comment]
+10k lines for supporting nothing more than a select box (MultiselectTags), less than 47 loc of own code. I love modern JS.

**Pull request closed without merging**